src/pages/DiaryHome/DiaryHome.js

```diff
@@ -12,39 +17,74 @@ import {
     FormControl,
     makeStyles,
     TextField,
-    Toolbar
+    Toolbar,
+    Typography
 } from '@material-ui/core';
+
+
 //import css files
 import '../../normalize.css'
 import './diaryPage.css'
 //Masonary css for grid
-
-const styles = makeStyles({
-    btn: {
-        color: "white",
-        border: "2px solid white",
-        '&:hover': {
-            background: "white",
-            color: "#34495e"
-        },
-    },
-    text: {
-        borderRadius: "10px",
-        borderWidth: "2px",
-        borderColor: '#3498db !important',
-    },
-    areaText: {
-        borderRadius: "10px",
-        borderWidth: "2px",
-        borderColor: '#3498db !important',
-    },
-    hidden: {
-        color: '#3498db'
-    }
-})
+import Masonry from 'react-masonry-css'
 
 function DiaryHome() {
 
+    const ref = firebase.firestore().collection("diary-notes")
+
+    ///States
+    const [diaryData, setDiaryData] = useState([]);
+
+    const [open, setOpen] = useState(false); ///using to handle dialog state
+
+    const [color, setColor] = useState({///using to handle form border colors(change to red when errors)
+        title: "#3498db",
+        description: "#3498db"
+    });
+
+    const [formData, setData] = useState({///using to handle form data
+        title: "",
+        description: ""
+    });
+
+    const [loading, isLoading] = useState(true)
+
+    ///Use-effect using
+    useEffect(() => {
```

Comment:
This is not required in this task. And this is not the way how it is done. This task is only for understanding how firebase works and configuring firestore in to project.